libevmcore/CommonSubexpressionEliminator.cpp

```diff
@@ -230,26 +343,40 @@ AssemblyItems CSECodeGenerator::generateCode(
 
 	// @todo: provide information about the positions of copies of class elements
 
-	// generate the dependency graph
+	// generate the dependency graph starting from final storage and memory writes and target stack contents
```

Comment:
Hm, but it should say "final storage _writes_, final memory writes and target stack contents".